test/adapter/json_api/transform_test.rb

```diff
@@ -102,8 +102,8 @@ def test_success_document_transform_global_config
           mock_request
           result = with_config(key_transform: :camel_lower) do
             serializer = PostSerializer.new(@post)
-            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer)
-            adapter.serializable_hash(@options)
+            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer, @options)
```

Comment:
I commented that out and it still failed. It appears to go away when I rm-rf test/action_controller